test/unit/lib/plugins/aws/provider.test.js

```diff
@@ -1197,6 +1197,30 @@ aws_secret_access_key = CUSTOMSECRET
       });
     });
 
+    describe('with `functions[].image` referencing existing images where ecr uri region is different from the provider region', () => {
```

Comment:
Let's avoid this describe as it only has one test case in it - let's put the `it` there directly (under existing `describe('with functions[].image referencing existing images'`